cns/singletenantcontroller/kubecontroller/crdrequestcontroller.go

```diff
@@ -116,20 +114,20 @@ func New(cfg Config) (*requestController, error) {
 	// for serving prometheus metrics, set to "0" to disable
 	mgr, err := ctrl.NewManager(cfg.KubeConfig, ctrl.Options{
 		Scheme:             scheme,
-		MetricsBindAddress: prometheusAddress,
+		MetricsBindAddress: ":9090",
```

Comment:
missed this before, but should this come from config?

Reply:
maybe. `:9090` is the standard prometheus scrape port, and i don't think we will have any reason to change it, but i'm not against making that a knob with `:9090` as an overridable default

Reply:
I think it would be better for orchestrator teams other than AKS (which may or may not run CNS as a container), such that they don't end up with any port collisions.

Reply:
i will add this as default so that it can be disabled with a config change.
just to note: as currently implemented, I'm using controller-runtime's prometheus integration so this codepath only gets exercised if we're running in k8s